.github/workflows/benchmark-parser.yml

```
@@ -19,7 +19,7 @@ jobs:
       MAIN-BENCH-20: ${{ steps.benchmark-main.outputs.BENCH_RESULT20 }}
```

This one is the same as `benchmark.yml`

```suggestion
      MAIN-BENCH-20: ${{ steps.benchmark-main.outputs.BENCH_RESULT20 }}
      MAIN-BENCH-22: ${{ steps.benchmark-main.outputs.BENCH_RESULT22 }}
```